cmake/opendds_build_helpers.cmake

```diff
@@ -18,6 +18,24 @@ function(_opendds_alias target)
     add_executable("${name}" ALIAS "${target}")
   endif()
   set_target_properties(${target} PROPERTIES EXPORT_NAME "${name}")
+
+  if(OPENDDS_COMPILE_WARNINGS STREQUAL "WARNING" OR OPENDDS_COMPILE_WARNINGS STREQUAL "ERROR")
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+      target_compile_options(${target} PRIVATE
+        -Wall -Wpedantic -Wno-unused -Wextra -Wcast-align -Wcast-qual -Wctor-dtor-privacy
+        -Wdisabled-optimization -Wformat=2 -Winit-self -Wmissing-include-dirs -Woverloaded-virtual
+        -Wredundant-decls -Wshadow -Wsign-conversion -Wstrict-overflow=5 -Wundef
```

Comment:
Instead of duplicating these, what about putting them in a separate file which is read by both CMake and configure?

Reply:
Like acetao.ini does now? I don't know if we want to put them there or not though.